controller/logs.go

```diff
@@ -0,0 +1,62 @@
+package controller
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strings"
+	"time"
+)
+
+func (c *Controller) GetActiveDeploymentLogs(ctx context.Context, numLines int32) error {
+	projectID, err := c.cfg.GetProject()
+	if err != nil {
+		return err
+	}
+	environmentID, err := c.cfg.GetEnvironment()
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Fetching latest deployment...")
```

Comment:
Should we print this or use the loading train indicator?

Reply:
I kinda like just mentioning what's actually going on

Reply:
Can use the train at the end maybe? It's REALLY quick tho

Reply:
Hmm, if its really quick is it needed? Might be weird for up if you `railway up` and then it shows `Fetching latest deployment`

Reply:
Eh that's true